src/engine.py

```
@@ -10548,25 +10549,272 @@ async def run_turn(transcript_text: str) -> None:
+                                            if tts_chunk:
+                                                if first_tts_ts is None:
+                                                    first_tts_ts = time.time()
+                                                    turn_latency_ms = (first_tts_ts - turn_start_time) * 1000
+                                                    session.turn_latencies_ms.append(turn_latency_ms)
+                                                    try:
+                                                        if t_start is not None:
+                                                            _TURN_STT_TO_TTS.labels(pipeline_label, provider_label).observe(
+                                                                max(0.0, first_tts_ts - t_start)
+                                                            )
+                                                    except Exception:
+                                                        pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)